makefile

```diff
@@ -252,8 +252,9 @@ Ccc  = $(strip $(CC) $(CFLAGS) $(GL_OPTS))
 ALL_SOURCE_FILES = \
  lib/gl/renderer.cpp lib/gl/renderer_core.cpp lib/gl/renderer_ff.cpp \
  lib/gl/shader.cpp lib/gl/types.cpp lib/aux_js.cpp lib/aux_vis.cpp \
- lib/font.cpp lib/gl2ps.c lib/gltf.cpp lib/material.cpp lib/openglvis.cpp \
- lib/palettes.cpp lib/palettes_base.cpp lib/sdl.cpp lib/sdl_helper.cpp lib/sdl_main.cpp \
+ lib/coll_reader.cpp lib/data_state.cpp lib/file_reader.cpp lib/font.cpp \
+ lib/gl2ps.c lib/gltf.cpp lib/material.cpp lib/openglvis.cpp lib/palettes.cpp \
+ lib/palettes_base.cpp lib/sdl.cpp lib/sdl_helper.cpp lib/sdl_main.cpp \
```

Comment:
heads-up that there are changes here in the release branch https://github.com/GLVis/glvis/pull/331/files#diff-beda42571c095172ab63437d050612a571d0d9ddd3ad4f2aecbce907a9b7e3d0